### PR TITLE
Rename unstable sync config

### PR DESCRIPTION
**Pull request by JonoPrest**

### CODE REVIEW

File: codegenerator/cli/src/config_parsing/human_config.rs [hunk 1]
```diff
@@ -361,7 +376,8 @@ pub mod evm {
         )]
         pub url: SingleOrList<String>,
         #[serde(skip_serializing_if = "Option::is_none")]
-        pub unstable__sync_config: Option<SyncConfigUnstable>,
+        #[schemars(description = "Config options for RPC syncing")]
+        pub sync_config: Option<RpcSyncConfig>,
```
Comment (DZakh) [inline codegenerator/cli/src/config_parsing/human_config.rs:380]: Feels like it should be flattend in the `RpcConfig`

File: codegenerator/cli/templates/static/codegen/src/Env.res [hunk 2]
```diff
@@ -107,13 +107,13 @@ module Configurable = {
...
+      envSafe->EnvSafe.get("RPC_SYNC_CONFIG_INITIAL_BLOCK_INTERVAL", S.option(S.int))
     let backoffMultiplicative =
-      envSafe->EnvSafe.get("UNSTABLE__SYNC_CONFIG_BACKOFF_MULTIPLICATIVE", S.option(S.float))
+      envSafe->EnvSafe.get("RPC_SYNC_CONFIG_BACKOFF_MULTIPLICATIVE", S.option(S.float))
     let accelerationAdditive =
-      envSafe->EnvSafe.get("UNSTABLE__SYNC_CONFIG_ACCELERATION_ADDITIVE", S.option(S.int))
+      envSafe->EnvSafe.get("RPC_SYNC_CONFIG_ACCELERATION_ADDITIVE", S.option(S.int))
     let intervalCeiling =
-      envSafe->EnvSafe.get("UNSTABLE__SYNC_CONFIG_INTERVAL_CEILING", S.option(S.int))
+      envSafe->EnvSafe.get("RPC_SYNC_CONFIG_INTERVAL_CEILING", S.option(S.int))
```
Comment (DZakh) [inline codegenerator/cli/templates/static/codegen/src/Env.res:116]: Let's prefix it with `ENVIO` and I think the `SYNC_CONFIG` part is not needed
```suggestion
      envSafe->EnvSafe.get("ENVIO_RPC_NTERVAL_CEILING", S.option(S.int))
```

**Review by DZakh — COMMENTED** [review]
It feels like from the user perspective that sync_config fields should be flattened in the RpcConfig

File: codegenerator/cli/templates/static/codegen/src/Env.res (hunk 2 above)
Comment (JonoPrest) [reply codegenerator/cli/templates/static/codegen/src/Env.res:116]: Ok cool 👍🏼

File: codegenerator/cli/src/config_parsing/human_config.rs (hunk 1 above)
Comment (JonoPrest) [reply codegenerator/cli/src/config_parsing/human_config.rs:380]: Ok sure we can do that 👍🏼

**Review by DZakh — APPROVED** [review]

**Pull request merged**